daml-foundations/daml-ghc/test-src/DA/Test/ShakeIdeClient.hs

```diff
@@ -306,15 +305,12 @@ goToDefinitionTests mbScenarioService = Tasty.testGroup "Go to definition tests"
             setFilesOfInterest [foo]
             expectNoErrors
             expectGoToDefinition (foo,2,[-1])   Missing             -- (out of range)
-            -- expectGoToDefinition (foo,2,[0..2]) (At (foo,3,0))   -- "foo"            [see failing test]
+            -- expectGoToDefinition (foo,2,[0..2]) (At (foo,3,0))   -- "foo" [see failing test "Go to definition takes type sig to definition"]
             expectGoToDefinition (foo,2,[2..4]) Missing             -- " : "
-            -- expectGoToDefinition (foo,2,[6..8]) (In "GHC.Types") -- "Int"            [see failing test]
```

Comment:
This has been fixed since, see the `Standard library type points to standard library` test which checks that we get `Missing` for types in `GHC.Types` (since it’s wired in).